Casks/h/hyper-connect.rb

```diff
@@ -0,0 +1,27 @@
+cask "hyper-connect" do
+  version "1.0.3"
+  sha256 :no_check
+
+  url "https://mijia-ios-adhoc.cnbj1.mi-fds.com/mijia-ios-adhoc/hyperconnect/HyperConnect.dmg",
+      verified: "mijia-ios-adhoc.cnbj1.mi-fds.com/mijia-ios-adhoc/hyperconnect/"
```

Comment:
```suggestion
  sha256 "c5ccba949f29f1fda54f9ebcf0f16a7c79e71a28732a50f4894406ef3a07727d"

  url "https://cdn.cnbj1.fds.api.mi-img.com/mijia-ios-adhoc/hyperconnect/HyperConnect-#{version}.dmg",
      verified: "mi-img.com/mijia-ios-adhoc/hyperconnect/"
```

We can add the versioned URL used by the in-app updater.

Reply:
There are no versions available for identification in PageHome.